Demo/app/build.gradle

```diff
@@ -32,13 +33,25 @@ android {
     kotlinOptions {
         jvmTarget = '17'
     }
+
+    buildFeatures {
+        viewBinding true
+        dataBinding true
+    }
 }
 
 dependencies {
-
+    def lifecycle_version = "2.6.1"
     implementation 'androidx.core:core-ktx:1.10.1'
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.9.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycle_version"
+    implementation 'androidx.fragment:fragment-ktx:1.5.7'
```

Comment:
Don't commit changes in this PR. Upgrades should be done in separate PR

Reply:
Is this resolved?

Reply:
Yes moved this to helper PR